javascript/cable_ready.js

```diff
@@ -172,9 +172,13 @@ const perform = operations => {
           } else {
             detail.element = document
           }
-          DOMOperations[name](detail)
+          if (detail.element || options.processMissing)
+            DOMOperations[name](detail)
```

Comment:
```js
if (!options.ignoreMissingElements) DOMOperations[name](detail)
```

The negated check doesn't read quite as well here, but I think the new option name and default value make more sense to consumers of the lib. What do you think?